typedoc.json

```diff
@@ -1,7 +1,6 @@
 {
-  "mode": "file",
   "out": "docs",
-  "theme": "gitbook",
+  "theme": "markdown",
```

Comment:
I don't know if it would work for our docs. Have you tried to generate them with this theme? If this typedoc version does not support gitbook integration, we should check how to solve it.